apps/docs/components/docs/toc.tsx

```diff
@@ -123,6 +135,35 @@ export const DocsToc: FC<DocsTocProps> = ({headings}) => {
           </a>
         </li>
       </ul>
+      <div className="w-full border border-default/60 hover:shadow-inner hover:border-default/80 rounded-md p-2 cursor-pointer">
+        <a
+          className="flex flex-col items-center"
+          href="https://nextui.pro?utm_source=nextui.org&utm_medium=callout"
```

Comment:
may double check if the link is expected to navigate to landing page or it should be component page. As a user, if I wanna see the component, I first click this banner, then navigate to landing page, then i need another click to see the components.

Reply:
updated to redirect to the components page.